components/transport-netty-axis2-bridge/src/main/java/org/wso2/transports/http/bridge/util/MessageUtils.java

```diff
@@ -123,4 +131,117 @@ public static boolean isEmptyPayloadStream(InputStream inputStream) throws IOExc
 
         return isEmptyPayload;
     }
+
+    /**
```

Comment:
Refer to https://www.oracle.com/technetwork/java/javase/tech/index-137868.html to learn about how to write javadocs.

Reply:
Will do.